server/routes/login/web-app.post.ts

```diff
@@ -103,5 +109,5 @@ export default eventHandler(async (event) => {
     id: _id,
   });
   save();
-  return await ModelUser.findOne({ _id });
+  return omit((await ModelUser.findOne({ _id })).toObject(), ["privateKey"]);
```

Comment:
Use flattenMaps, because meta will not be transformed to object
```suggestion
  return omit((await ModelUser.findOne({ _id })).toObject({ flattenMaps: true }), ["privateKey"]);
```